### DISCUSSION TITLE
[Build] 대 소문자 변경을 인식하지못하여 git Action CI Build Fail

### DISCUSSION THREAD

**Discussion by ngsy** [GymHubCommunity/GymHub-FE#57] [ErrorLog] [answered]
## 개요
카멜 케이스를 파스칼 케이스로 변경함으로써 파일명 첫 글자를 대문자로 변경 후 에러 발생 

## 특이사항
로컬에서 build할땐 문제없이 통과됨 그러나 git actions CI를 통한 build는 에러 발생

![image](https://github.com/GymHubCommunity/GymHub-FE/assets/72487120/30cdb15d-746f-4288-a2ce-dcd46eca0b7c)

**Comment by ngsy** [comment, marked as answer]
### 해결 방법

1. 로컬과 동일한 환경 만들기
  현재 로컬에선 대소문자를 구분하기 위해 ignorecase 옵션을 꺼놨습니다.
  그래서 git actions CI 환경에도 해당 커멘드를 추가해주었습니다.
 ```
     - name: Configure Git
        run: git config core.ignorecase false
 ```
옵션을 추가 해주었음에도 동일한 에러가 발생하였습니다.

2. 로컬 git cache 삭제
로컬 cache에 변경 전에 파일이 남아있는것으로 보여 하위 커멘드를 사용하여 로컬 앱 캐시를 삭제 하고 삭제된 이력을 다시 push하였습니다.
   ```
         git rm -r --cached .
         git add .
         git commit -m "commit message"
         git push 
   ```

그 결과  성공!
<img width="729" alt="image" src="https://github.com/GymHubCommunity/GymHub-FE/assets/72487120/919e82a4-7f12-48d2-b2b5-fe1e13d1927e">

**Comment by oauch** [comment]
덕분에 git에 대해서 배워갑니다 👍🏻
